### PR TITLE
File upload front

### PR DESCRIPTION
**Pull request by soupette**
🔥 Upload files int the Admin 🔥

Created Upload plugin
Add list view to see, search, upload, delete files

Remove useless components and containers in the content manager

### CODE REVIEW

**Comment by lauriejim** [comment]
LGTM
But before validate this PR, you comment many lines of code.
Is it useful to kept them ?

**Comment by soupette** [comment]
Yes for the time being!

File: packages/strapi-plugin-upload/admin/src/translations/en.json [hunk 1]
```diff
@@ -1,4 +1,6 @@
 {
+  "EmptyLi.message": "There is no uploaded files",
```
Comment (Aurelsicoko) [inline packages/strapi-plugin-upload/admin/src/translations/en.json]: `There are...`

File: packages/strapi-plugin-upload/admin/src/translations/de.json [hunk 2]
```diff
@@ -1,4 +1,6 @@
 {
+  "EmptyLi.message": "There is no uploaded files",
```
Comment (Aurelsicoko) [inline packages/strapi-plugin-upload/admin/src/translations/de.json]: `There are...`

File: packages/strapi-plugin-upload/admin/src/translations/pl.json [hunk 3]
```diff
@@ -1,4 +1,6 @@
 {
+  "EmptyLi.message": "There is no uploaded files",
```
Comment (Aurelsicoko) [inline packages/strapi-plugin-upload/admin/src/translations/pl.json]: Same.

**Pull request merged**